django/contrib/gis/db/backends/postgis/adapter.py

```diff
@@ -51,13 +53,16 @@ def prepare(self, conn):
         This method allows escaping the binary in the style required by the
         server's `standard_conforming_string` setting.
         """
-        if self.is_geometry:
+        if self.is_geometry and self.ewkb:
             self._adapter.prepare(conn)
 
     def getquoted(self):
         """
         Return a properly quoted string for use in PostgreSQL/PostGIS.
         """
+        if not self.ewkb:
+            return "ST_GeomFromText('POINT EMPTY', %s)" % self.srid
```

Comment:
There's other types of empty geometry, shouldn't this be type-specific? (ala `GEOMETRYCOLLECTION EMPTY`, `POLYGON EMPTY`, etc)